x-pack/plugins/security_solution/public/cloud_security_posture/components/misconfiguration/misconfiguration_preview.tsx

```diff
@@ -136,6 +109,7 @@ export const MisconfigurationsPreview = ({
     sort: [],
     enabled: true,
     pageSize: 1,
+    ignore_unavailable: false,
```

Comment:
can you add a comment explaining the reasoning behind this?

Reply:
woops, this shouldnt be false, it should be true
reasoning behind this ignore_unavailable is basically to prevent error from showing up when we don't have findings index when opening host.name or user.name